qa/sidecar_grid_check.py

```
@@ -0,0 +1,115 @@
+            print(f"{room}: SKIP (no sidecar at {boxes_path})")
+            continue
+        res = check(json.loads(geof.read_text()), json.loads(boxes_path.read_text()))
+        bwo, oof = res["blocked_without_occluder"], res["occluder_over_open_floor"]
+        verdict = "OK" if not bwo and not oof else "DISAGREE"
+        print(f"{room}: {verdict} boxes={res['boxes']} blocked-without-occluder {len(bwo)} {bwo[:12]}{'…' if len(bwo) > 12 else ''}"
+              f" occluder-over-open-floor {len(oof)} {oof[:12]}{'…' if len(oof) > 12 else ''}  [{boxes_path.name}]")
+        checked += 1
+        if verdict != "OK":
+            worst = 1
+    if not checked:
+        print("nothing checkable"); return 2
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)